faqs/galaxy/Datasets_Data_Privacy_Features.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Data Privacy Features 
+area: histories
+box_type: tip
+layout: faq
+contributors: [desmond-jasper]
+---
+
+Privacy controls are only enabled if desired. Otherwise, datasets by defaults remain private and unlisted in Galaxy.. 
+This means that a dataset you've created is virtually invisible until you publish a link to it. 
```

Comment:
```suggestion
Privacy controls are only enabled if desired. Otherwise, datasets by defaults remain private and unlisted in Galaxy. This means that a dataset you've created is virtually invisible until you publish a link to it. 
```